_includes/api-member.html

```diff
@@ -6,6 +6,14 @@ <h2><a class="anchorable" href="#extends">Extends</a></h2>
   {% endfor %}
 {% endif %}
 
+{% if include.data.summary.size > 0 %}
```

Comment:
Why is this below the extends part? I would expect the summary to be first, with additional details below that.

Reply:
I didn't know what extends was, that's all. Will move it up.

[ ] Move

Reply:
It just basically says something like this:
```
Extends
Observable
```
I assume we don't use this, as it's horrible that isn't a list :facepalm:

Reply:
...I've seen the tags used in a couple places, will investigate

Reply:
I don't think borrows/Extends is working anyway.